### DISCUSSION TITLE
Better tools for docs or private GitHub pages

### DISCUSSION THREAD

**Discussion by HonzaMikula** [community/community#4731] [Other Feature Feedback, Questions, & Ideas]
In recent years we move everything around developing apps directly inside GitHub. Tools like Projects are really great. The last missing piece is to have some reasonable way how to manage documentation for projects. 

Currently, you can use GitHub Wiki. But that solution is not usable. Even trivial thing like upload image is basically impossible. 

Second option we investigated is using GitHub pages in combination with tools like https://docsify.js.org. Trouble is that GitHub Pages are always public and we don't want to have our documentation public.

Are there some plans to improve this area in GitHub? Thanks.

**Comment by PabloLION** [comment]
I hope they can implement GitHub CodeSpace/ VSCode online (github.dev editor). Thus everyone can edit the markdown file with his/her own favorite extension.
